### ISSUE TITLE
Would be great to have the possibility to initialize the node(s) in a sandbox

### ISSUE THREAD

**Issue by lefred** [datacharmer/dbdeployer#123]
**Describe the solution you'd like**
With MySQL 8.0 allowing to change some configuration settings directly in the server (stored in datadir), I think it would be great when for testing purpose to have the possibility to initialize again the nodes in a sandbox.

**Describe alternatives you've considered**
Making my own script ;)

**Additional context**
So the script would stop all nodes (or the selected one), remove the datadir and initialize again

**Comment by lefred** [comment]
This is the content of my current `initialize_all` script:

```
./stop_all
for node in $(ls -d node*)
do
        rm -rf $node/data/*
        cd $node
        ./init_db
        cd -
done
```

**Comment by datacharmer** [comment]
That behaviour is easy to reproduce if we want to re-initialize all the nodes (whether in regular replication or group topology), but  not for a single node.

Notice that your script is not enough to resume functionality, though. For each node we also need to run `start` and `load_grants`, before eventually run `./initialize_nodes`
(See: https://github.com/datacharmer/dbdeployer/wiki/skip-server-start) 

If this is what you need, it would be easy to add a command `wipe_and_restart`.

**Comment by lefred** [comment]
That would be great and useful. I'm now doing several tests and I need to reset multiple times

**Comment by lefred** [comment]
and yes, single node could be useful too even if not really in my case, but I see it could be nice for other cases

**Comment by datacharmer** [comment]
While a complete reset would be possible with minimum effort, a single node reset requires a rewrite of the `initialize_nodes` in a distributed way. I will think about it

**Comment by datacharmer** [comment]
Implemented in version 1.58.0.
See [Documentation](https://github.com/datacharmer/dbdeployer/wiki/main-operations#re-deploy-a-sandbox)
